microsoft-edge/devtools-guide-chromium/progressive-web-apps/simulate-window-controls-overlay.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Simulate PWA Window Controls Overlay
+description: Simulate PWA Window Controls Overlay.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.subservice: devtools
+ms.date: 05/13/2024
+---
+# Simulate PWA Window Controls Overlay
+
+In DevTools, in the **Application** tool, you can simulate the Window Controls Overlay feature that Progressive Web Apps (PWAs) have, without having to install the PWA.  The **Application** tool has a **Window Controls Overlay** section under **Manifest**, to display a preview of how your Progressive Web App (PWA) handles window controls, without having to install the PWA and make the styling changes that are necessary to accommodate window controls:
+
+![The 'Window Overlay Controls' section in the Application tool](./simulate-window-controls-overlay-images/pwa-window-controls.png)
+
+1. In the manifest file, set the `display-override` field to `window-controls-overlay`:
+
+   ```json
+   {
+     "display_override": ["window-controls-overlay"]
+   }
+   ```
+
+   For more information, see [Enable the Window Controls Overlay in your app](../../progressive-web-apps-chromium/how-to/window-controls-overlay.md#enable-the-window-controls-overlay-in-your-app) in _Display content in the title bar_.
+
+1. In Microsoft Edge, select **Settings and more** > **More tools** > **Developer tools**.  Or, press **Ctrl+Shift+I** (Windows, Linux) or **Command+Option+I** (macOS).  DevTools opens.
+
+1. On the **Activity Bar**, select  the **Application** tab.  If that tab isn't visible, click the **More tools** (![More tools icon](./simulate-window-controls-overlay-images/more-tools-icon.png)) button.
+
+1. In the **Application** tool, expand **Manifest** in the upper left, and then select **Window Controls Overlay**.
+
+1. In the **Window Controls Overlay** section on the right, select the checkbox **Emulate the Window Controls Overlay on** \<platform\>, and select a platform, such as Windows.
+
+<!-- todo: then what?  what action, what result?
+1. 
+-->
```

Comment:
I think we need to describe the result of selecting the checkbox, and we need to add a screenshot showing the result:

```suggestion
1. In the **Window Controls Overlay** section on the right, select the checkbox **Emulate the Window Controls Overlay on** \<platform\>, and select a platform, such as Windows.

   The rendered web page now simulates the WCO feature:
   * An overlay that shows the window controls appears.
   * The values of the CSS `env(titlebar-area-left)`, `env(titlebar-area-top)`, `env(titlebar-area-width)`, and `env(titlebar-area-height)` variables are set to match the platform you selected.
```

Reply:
added:  todo: screenshot showing result